plugin-server/src/utils/db/db.ts

```diff
@@ -126,10 +126,12 @@ export interface CreatePersonalApiKeyPayload {
     created_at: Date
 }
 
-type PersonData = {
+export interface CachedPersonData {
     uuid: string
-    created_at_iso: string
+    createdAtIso: string
```

Comment:
This and `team_id` are inconsistent now, based on the convention in types maybe change this back?

Reply:
will standardize